src/Umbraco.Web/Editors/ContentController.cs

```diff
@@ -1736,12 +1737,18 @@ private void MapValuesForPersistence(ContentItemSave contentSave)
             bool Varies(Property property) => property.PropertyType.VariesByCulture();
 
             var variantIndex = 0;
-
+            var newContent = (contentSave.Action == ContentSaveAction.SaveNew
```

Comment:
IIRC there's a method on this controller already to detect if the action is "New" which can be re-used instead